pyomo/core/tests/unit/test_con.py

```diff
@@ -84,21 +84,55 @@ def rule(model):
         self.assertEqual(model.c.upper, 0)
 
     def test_tuple_construct_inf_equality(self):
-        model = self.create_model(abstract=True)
-
-        def rule(model):
-            return (model.x, float('inf'))
-
-        model.c = Constraint(rule=rule)
-        self.assertRaises(ValueError, model.create_instance)
-
-        model = self.create_model(abstract=True)
-
-        def rule(model):
-            return (float('inf'), model.x)
+        model = self.create_model(abstract=True).create_instance()
```

Comment:
Why not change this to just create a `ConcreteModel`? Are there behaviors we expect with `AbstractModels` that are no longer being tested because of this change?

Reply:
I think we *could* just switch this over to a ConcreteModel.  I left it as it was mostly to minimize changes in old code / tests.  The biggest reason for the change here was to remove the test for an exception on model creation, as this PR defers the bounds sanity checking until later (model compilation step and not model construction step).